src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -76,7 +78,15 @@ public InventoryAccountUsageCollector(
 
   @SuppressWarnings("squid:S3776")
   @Transactional
-  public Map<String, AccountUsageCalculation> collect(Collection<String> products, String account) {
+  public Map<String, AccountUsageCalculation> collect(
+      Collection<String> products, AccountConfig accountConfig) {
+    String account = accountConfig.getAccountNumber();
+    String orgId = accountConfig.getOrgId();
+    if (Objects.isNull(account) || Objects.isNull(orgId)) {
+      throw new IllegalArgumentException(
+          String.format("Incomplete opt-in configuration - account: %s orgId: %s", account, orgId));
```

Comment:
```suggestion
          String.format("Incomplete opt-in configuration - account=%s orgId=%s", account, orgId));
```
is more searchable via splunk.

Reply:
DONE